sympy/parsing/tests/test_latex.py

```diff
@@ -260,28 +260,3 @@ def test_failing_not_parseable():
     for latex_str in FAILING_BAD_STRINGS:
         with raises(LaTeXParsingError):
             parse_latex(latex_str)
-
-
-def test_antlr_generation():
```

Comment:
Why are you removing a test?

Reply:
The test runs antlr and checks if the files are the same as the existing ones in the _antlr
directory.
Now that the .interp and .tokens files have been removed it doesn't work.
We could still make it check for the listener, parser and lexer files but
the test as a whole seems unecessary.